daemons/controld/controld_control.c

```diff
@@ -44,7 +44,7 @@ do_ha_control(long long action,
     static crm_cluster_t *cluster = NULL;
 
     if (cluster == NULL) {
-        cluster = calloc(1, sizeof(crm_cluster_t));
+        cluster = pcmk_cluster_new();
```

Comment:
No free?

Reply:
It never has been freed here and I'm not sure how it could be without some restructuring, since it's function-static. We have an open task for restructuring this: https://projects.clusterlabs.org/T598

Reply:
That makes sense, we can leave it for later